fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -11,6 +11,19 @@ def all(requested_os_type = "")
         return connected_devices(requested_os_type) + simulators(requested_os_type)
       end
 
+      def runtime_build_os_versions
+        @runtime_build_os_versions ||= begin
+            output, status = Open3.capture2('xcrun simctl list runtimes -j')
+            raise status unless status.success?
+            json = JSON.parse(output)
+            json['runtimes'].map { |h| [h['buildversion'], h['version']] }.to_h
+          rescue StandardError => e
+            UI.error(e)
+            UI.error('xcrun simctl CLI broken; cun `xcrun simctl list runtimes` and make sure it works')
+            UI.user_error!('xcrun simctl not working')
+          end
```

Comment:
Nitpick (that our rubocop should warn about): inconsistent indentation there, should be two spaces less indented

Reply:
Odd, my Rubocop didn't pick up on this. Went through the changes again to fix indentation. Should be good to go now.